.github/website/src/scripts/theme-toggle.js

```
@@ -33,14 +53,38 @@ function updateAllIcons() {
+    });
+  } catch (e) {
+    // Emit: website.theme.storage.failure
+    telemetry.emit('website.theme.storage.failure', {
+      safe: {
+        failureType: e.name || 'StorageError',
+        theme: toTheme,
+        fallbackUsed: false
+      },
+      restricted: {
+        storageError: e.message,
+        browserInfo: navigator.userAgent
```

ESLint

[error] 'navigator' is not defined.

(no-undef)